kie-server-parent/kie-server-api/src/test/java/org/kie/server/api/marshalling/objects/DateObject.java

```diff
@@ -0,0 +1,62 @@
+package org.kie.server.api.marshalling.objects;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlRootElement;
+import javax.xml.bind.annotation.adapters.XmlJavaTypeAdapter;
+
+import com.thoughtworks.xstream.annotations.XStreamAlias;
+import com.thoughtworks.xstream.annotations.XStreamConverter;
+import org.kie.internal.jaxb.LocalDateTimeXmlAdapter;
```

Comment:
This is bad (I hadn't noticed that before).
We shouldn't be writing our own JAXB adapters and XStream converters for LocalDate and other java.time.*  classes.
Those already exist, hopefully written and maintained by the JAXB and XStream guys themselves. But they might be in a extension jar (= separate dependency) because of JDK 6/7 compatibility issues.